src/bluesearch/database/topic.py

```diff
@@ -295,3 +295,45 @@ def get_topics_for_pmc_article(
             journal_topics.append(descriptor["name"])
 
     return journal_topics
+
+
+def get_topics_for_pubmed_article(
+    article: Element,
+) -> Tuple[Optional[list[str]], Optional[list[str]]]:
+    """Extract journal topics of a PubMed article.
+
+    Parameters
+    ----------
+    article
+        Article for which to extract journal and article topics.
+
+    Returns
+    -------
+    journal_topics : Optional[list[str]
```

Comment:
Square bracket missing

Reply:
Change for `list[str] | None` in 37df3a6.